project/Settings.scala

```diff
@@ -56,8 +56,7 @@ object Settings {
       "-feature",
       "-deprecation",
       "-unchecked",
-      "-Xlint",
-      "-Xfatal-warnings"
```

Comment:
Me too, but for now we can't have it if we want to use DB.withTransaction and DB.withSession which are deprecated.